mycroft/messagebus/message.py

```diff
@@ -36,10 +55,42 @@ def serialize(self):
 
     @staticmethod
     def deserialize(value):
+        """This takes a string and constructs a message object.
+
+        This makes it easy to take strings from the websocket and create
```

Comment:
This comment crosses the line of going into implementation details too deeply. The goal of docs should be to be useful, but not out of date as soon as the code is change (I could, for example, change this to use ujson or another json library). 

What's helfpful here is that value is a json string, matching the serialized Message format. Other details are extraneous.